### DISCUSSION TITLE
CORS problem.

### DISCUSSION THREAD

**Discussion by mzch** [umputun/remark42#1534] [Q&A] [answered]
I installed remark42 with `uixis.org` domain and I tried to use it from`mzch.org`, but `localStorage access denied` error happend.
I changed Chrome Cookie setting and it was resolved but `GET /api/v1/user` call returned 401 error.

<img width="1476" alt="ScreenShot" src="https://user-images.githubusercontent.com/621178/202904231-ccc53919-8f25-4bbb-85ca-1c2824d2c006.png">

- remark42 log
```
Nov 20 21:38:50 ls remark42.linux-amd64[2233730]: 2022/11/20 21:38:50.513 [INFO]  GET - /api/v1/find?site=mzchremark&url=https://mzch.org/2022/10/12/0024-macstudio/&sort=-active&format=tree - uixis.org - c005266b2c90 - 200 (115) - 531.969µs
Nov 20 21:39:03 ls remark42.linux-amd64[2233730]: 2022/11/20 21:39:03.957 [INFO]  GET - /auth/github/login?from=https://uixis.org/web/iframe.html?selfClose&site=mzchremark - uixis.org - c005266b2c90 - 302 (236) - 945.106µs
Nov 20 21:39:05 ls remark42.linux-amd64[2233730]: 2022/11/20 21:39:05.244 [INFO]  GET - /auth/github/callback?code=33d2f9a66df21c724d5f&state=c84a35d24f51a3ed6813ed705e57669a589144a9 - uixis.org - c005266b2c90 - 307 (79) - 578.75992ms
Nov 20 21:40:17 ls remark42.linux-amd64[2233730]: 2022/11/20 21:40:17.208 [INFO]  GET - /api/v1/config?site=mzchremark - uixis.org - c005266b2c90 - 200 (822) - 372.377µs
Nov 20 21:40:17 ls remark42.linux-amd64[2233730]: 2022/11/20 21:40:17.345 [INFO]  GET - /api/v1/find?site=mzchremark&url=https://mzch.org/2022/10/12/0024-macstudio/&sort=-active&format=tree - uixis.org - c005266b2c90 - 200 (115) - 249.799µs
Nov 20 21:40:22 ls remark42.linux-amd64[2233730]: 2022/11/20 21:40:22.353 [INFO]  GET - /auth/github/login?from=https://uixis.org/web/iframe.html?selfClose&site=mzchremark - uixis.org - c005266b2c90 - 302 (236) - 508.922µs
Nov 20 21:40:23 ls remark42.linux-amd64[2233730]: 2022/11/20 21:40:23.272 [INFO]  GET - /auth/github/callback?code=40c43564354139a6d177&state=6d1eeec2872e1f317bfec6b54053e5baf05434cb - uixis.org - c005266b2c90 - 307 (79) - 438.951033ms
```

Finally it's fixed when I put remark42 into `mzch.org` subdomain, but I think it's not smart. Could anyone tell me how to run remark42 on the different domain?

OS: macOS `13.0.1`
Brwoser: Google Chrome `107.0.5304.110`

**Comment by paskal** [comment, marked as answer]
Have you read the appropriate [documentation page](https://remark42.com/docs/manuals/separate-domain/)?

**Reply by mzch** [reply]
Yes, I set `ALLOWED_HOSTS="mzch.org"`, but it happened.

**Reply by paskal** [reply]
Then could you please write what you are trying to do and what you are getting? 401 on the user credentials retrieve an expected code now for unauthorised users.
For authorised users, it shouldn't be the case if you're using one of the auth methods available from a separate domain name, but you haven't mentioned if you tried to log in and which login methods you tested.

**Reply by paskal** [reply]
I've missed it, and it's here in the logs: you're using GitHub as a login method. The page I linked explicitly says that it doesn't work when remark42 runs on a separate domain now.

**Reply by mzch** [reply]
Oh...Thanks for your advice!

**Comment by mzch** [comment]
I use Google Chrome, `107.0.5304.110`
